foundations/html_css/css-foundations/the-cascade.md

```diff
@@ -26,7 +26,7 @@ A CSS declaration that is more specific will take precedence over less specific
 2. Class selectors
 3. Type selectors
 
-Specificity will only be taken into account when an element has multiple, conflicting declarations targeting it, sort of like a tie-breaker. An ID selector will always beat any number of class selectors, <span id="high-specificity-class-type">a class selector will always beat any number of type selectors</span>, and a type selector will always beat any number of anything less specific than it. When no declaration has a selector with a higher specificity, a larger amount of a single selector will beat a smaller amount of that same selector.
+Specificity will only be taken into account when an element has multiple, conflicting declarations targeting it, sort of like a tie-breaker. An ID selector will always beat any number of class selectors, <span id="high-specificity-class-type">a class selector will always beat any number of type selectors</span>, and a type selector will always beat any number of less specific selectors. When no declaration has a selector with a higher specificity, a greater specificity of a single selector takes precedence over a lesser specificity of the same selector.
```

Comment:
Can you explain what this means with an example please @odilsonjs

"When no declaration has a selector with a higher specificity, a greater specificity of a single selector takes precedence over a lesser specificity of the same selector."

Reply:
Certainly! Let's illustrate this concept with a simple example in CSS:

Suppose you have the following CSS rules:
`/* Rule 1 */
p {
  color: red;
}

/* Rule 2 */
#special-paragraph {
  color: blue;
}
`
Now, you have an HTML document like this:
`<p id="special-paragraph">This is a special paragraph.</p>
`
In this example:

1.     The selector p in "Rule 1" has a lesser specificity because it's a type selector.
2.     The selector #special-paragraph in "Rule 2" has a greater specificity because it's an ID selector.

Since both selectors target the same <p> element, specificity becomes crucial in determining which style takes precedence. In this case:

-     The "greater specificity" (ID selector) from "Rule 2" takes precedence over the "lesser specificity" (type selector) from "Rule 1."

As a result, the text in the paragraph will be displayed in blue, following the style defined by the ID selector in "Rule 2."

This example demonstrates how the specificity of selectors determines which styles are applied when there are conflicting CSS rules.

Reply:
Thanks @odilsonjs, the first part of this paragraph is indeed about the rules you've just explained. However, the part you've changed is not. It's about when you are using the same kind of selectors.

For reference, the original text:

"When no declaration has a selector with a higher specificity, a larger amount of a single selector will beat a smaller amount of that same selector."

For example, when you are using class selectors, the declaration that uses more class selectors beats any that have less:

```css
.list .item {
...
}
```

has a higher specificity than

```css
.item {
...
}
```

The example that comes after this paragraph in the lesson demonstrates it.

I don't disagree the original line is confusing and wordy. But I'd argue the replacement text being proposed here is even more confusing.

Reply:
Hmmm... okay @KevinMulhern, Thank you for your clarification

After some reflection, i finally get your point of view.

No changes is required about it

Let's keep this as it is.

Reply:
That sounds good to me too @odilsonjs - I can't think of a better alternative right now. Lets get this sentence reverted to the older version and then we'll merge

Reply:
Okay, it's done!

Reply:
Hello @KevinMulhern, this PR is still open, do you need some more changes before Merge?